src/drive/web/modules/filelist/File.jsx

```diff
@@ -278,9 +277,7 @@ class File extends Component {
   open(e, attributes) {
     e.stopPropagation()
     if (isDirectory(attributes)) {
-      this.props.onFolderOpen(attributes.id).then(() => {
-        this.props.router.push(getFolderUrl(attributes.id, this.props.location))
-      })
+      this.props.onFolderOpen(attributes.id)
```

Comment:
Since we don't want to always use the URL to navigate, the calls to router have been moved to a component higher up the chain.